cmd/disk-cache-backend.go

```diff
@@ -164,6 +170,8 @@ func newDiskCache(ctx context.Context, dir string, config cache.Config) (*diskCa
 		lowWatermark:  config.WatermarkLow,
 		highWatermark: config.WatermarkHigh,
 		enableRange:   config.Range,
+		commitAsync:   config.CommitAsync,
+		retryAsyncCh:  make(chan ObjectInfo, 10000),
```

Comment:
What happens when this is full, will uploads block?

Reply:
yes - this happens only after server restart to pick up any pending/failed write backs.